src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1262,8 +1262,23 @@ private CrefParameterSyntax ParseCrefParameter()
                     break;
             }
 
+            SyntaxToken readOnlyOpt = null;
+            if (CurrentToken.Kind == SyntaxKind.ReadOnlyKeyword && refKindOpt is not null)
+            {
+                if (refKindOpt.Kind != SyntaxKind.RefKeyword)
+                {
+                    // if we encounter `readonly` after `in` or `out`, we place the `readonly` as skipped trivia on the previous keyword
+                    var misplacedToken = AddError(EatToken(), ErrorCode.ERR_RefReadOnlyWrongOrdering);
```

Comment:
>ERR_RefReadOnlyWrongOrdering [](http://example.com/codeflow?start=72&length=28)

We should report this as a warning. See how we doo that for feature availability check. Opened #69142 for ERR_MisplacedUnchecked.